Packages/Tracking Preview/PhysicsHands/Runtime/PhysicsGraspHelper.cs

```diff
@@ -502,63 +610,66 @@ private void UpdateRemovedBones()
 
         private void UpdateGraspingValues()
         {
```

Comment:
can we get a couple of comments in here to explain what's happening? There's a lot of checks

```suggestion
        {
        // Check if this hand was grasping and is now not grasping
```

Reply:
sure - I've asked @rorygames to comment this :)